contribs/gmf/src/directives/filterselector.js

```diff
@@ -266,6 +274,14 @@ gmf.FilterselectorController = class {
     } else {
       this.filtrableLayerNodeNames_ = null;
     }
+    if (usrFunc &&
+        usrFunc['preset_layer_filter'] &&
+        usrFunc['preset_layer_filter'][0]
+    ) {
+      this.defaultFiltrableDataSourceName_ = usrFunc['preset_layer_filter'][0];
+    } else {
+      this.defaultFiltrableDataSourceName_ = undefined;
+    }
```

Comment:
That would set a `false` value.  I want an `undefined` value.  Anyway, having a simplified version of this is not that useful since it already gets minimized and simplified automatically when building the minimized version.  I'll leave this as it is.